src/Calculator/Controls/MathRichEditBox.cpp

```diff
@@ -112,6 +113,16 @@ void MathRichEditBox::SetMathTextProperty(String ^ newValue)
     this->IsReadOnly = readOnlyState;
 }
 
+void MathRichEditBox::OnGotFocus(Platform::Object ^ sender, Windows::UI::Xaml::RoutedEventArgs ^ e)
+{
+    // [BUG 28498627][GithubIssue #1380]
+    // below directives on Selection are going to engage Narrator to announce the content of this richedit
+    // this is a workaround since richedit doesn't announce its content automatically.
+    this->Document->Selection->EndKey(TextRangeUnit::Line, false);
+    this->Document->Selection->HomeKey(TextRangeUnit::Line, false);
+    this->Document->Selection->MoveLeft(TextRangeUnit::Character, 1, false);
```

Comment:
This seems like it would be annoying for the end user if they click at a specific character they want to edit but the cursor is suddenly moved to a different location.